roboteam_ai/src/skills/Dribble.cpp

```diff
@@ -101,17 +99,14 @@ void Dribble::onInitialize() {
 }
 
 Dribble::Status Dribble::onUpdate() {
-
-    ball = World::getBall(); //TODO: sanity checking if ball is actually there?
-
-    if (currentProgress == Progression::FAIL) {
+    if (currentProgress == Progression::FAIL || !ball) {
```

Comment:
This is actually undesirable behavior as we saw in the designlab, as we want to robot to keep dribbling if it is in front of the ball from the camera's angle.